tensorflow_encrypted/protocol/pond.py

```diff
@@ -467,9 +469,8 @@ def conv2d_bw(self, x, w, strides, padding):
 
         return z
 
-#
+
```

Comment:
replace `#` with `#####` instead of empty lines, to make it easier to spot in the text file (single line comment are easier to miss)?